shanoir-ng-studies/src/main/java/org/shanoir/ng/subjectstudy/dto/SubjectStudyDTO.java

```diff
@@ -33,6 +33,8 @@ public class SubjectStudyDTO {
 
 	private IdName subject;
 
+	private boolean subjectPreclinical;
```

Comment:
Is this in DB ? If yes, I think we need a SQL script ! (but I'm not sure you actually want it in DB ?)